.circleci/config.yml

```diff
@@ -163,9 +163,22 @@ references:
         aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/
         aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/epoch-latest-${PKG_SUFFIX:?}.tar.gz
 
+  import_secrets: &import_secrets
+    run:
+      name: Import Secrets
+      command: |
+        echo "export AWS_CREDS_ROLE=epoch-inventory" >> $BASH_ENV
+        echo "source /infrastructure/import-secrets.sh" >> $BASH_ENV
```

Comment:
This is basically re-running the infrastructure Docker image entrypoint with a distinct AWS_CREDS_ROLE. Ok.

Not sure whether [setting var](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-container) in [job host config](https://github.com/aeternity/epoch/blob/b05808ab1955472646a25fdc6021da7b04452fc0/.circleci/config.yml#L604) would work. Anyway sleep would be needed still. And it is debatable. Ok.

Reply:
> Not sure whether [setting var](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-container) in [job host config](https://github.com/aeternity/epoch/blob/b05808ab1955472646a25fdc6021da7b04452fc0/.circleci/config.yml#L604) would work.

Yes, it would work, but why setting the same var multiple times (maintenance) ? It would also work to set it in the CircleCI environment variables configuration, however I would like to keep it in the repo/config (more explicit).